### PR TITLE
f-metadata@1.0.4 - Ensure that `handleContentCards` callback has been set before attempting to call it.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Fixed
- Ensure that `handleContentCards` callback has been set before attempting to call it.

### CODE REVIEW

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/f-metadata/CHANGELOG.md [hunk 1]
```diff
@@ -3,13 +3,21 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+1.0.4
+------------------------------
+*February  29, 2020*
```
Comment (ashleynolan) [inline packages/f-metadata/CHANGELOG.md]: Need some sort of time travel emoji here 😆

File: packages/f-metadata/CHANGELOG.md [hunk 2]
```diff
@@ -3,13 +3,21 @@
...
+
+### Fixed
+- Ensure that `handleContentCards` callback has been set before attempting to call it.
+
```
Comment (ashleynolan) [inline packages/f-metadata/CHANGELOG.md:12]: Super NIT: Need a double line break here

**Review by ashleynolan — COMMENTED** [review]
All good - just small CHANGELOG update needed

File: packages/f-metadata/CHANGELOG.md (hunk 1 above)
Comment (DamianMullins) [reply packages/f-metadata/CHANGELOG.md]: Ah bugger haha

Comment (DamianMullins) [reply packages/f-metadata/CHANGELOG.md]: At least there is a 29th Feb this year...

**Review by ashleynolan — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**